scipy/stats/_multivariate.py

```diff
@@ -247,18 +250,19 @@ def random_state(self, seed):
 _mvn_doc_default_callparams = """\
 mean : array_like, default: ``[0]``
     Mean of the distribution.
-cov : array_like, default: ``[1]``
+cov : array_like or `Covariance`, default: ``[1]``
     Symmetric positive (semi)definite covariance matrix of the distribution.
 allow_singular : bool, default: ``False``
-    Whether to allow a singular covariance matrix.
+    Whether to allow a singular covariance matrix. This is ignored if `cov` is
+    a `Covariance` object.
```

Comment:
Souldn't `` `Covariance` `` be changed to `` `CovViaPrecision` `` since that's the only class visible to the user?
```suggestion
    a `CovViaPrecision` object.
```

Reply:
I really hope that more `Covariance` objects will be available before the next release. The code is ready, anyway.

Reply:
Oh, OK. Makes sense.